addon/components/o-s-s/social-media-badge.hbs

```diff
@@ -0,0 +1,8 @@
+<div class={{this.computedClasses}} role="button"
+     {{enable-tooltip title=(t (concat "oss-components.post_types." @postType))}}
+     {{on "click" this.onPostTypeClick}}
+     ...attributes>
+    <div class="icon-container">
```

Comment:
I think your IDE config is wrong, this looks like '4 chars' for spaces but we use 2 chars.
Also the alignment of the div attributes seems weird

Reply:
Don't forget to add an empty line at the end of the file too